sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -744,6 +744,12 @@ def translate_distribution(distribution_update, metric_update_proto):
   dist_update_proto.max = to_split_int(distribution_update.max)
   dist_update_proto.count = to_split_int(distribution_update.count)
   dist_update_proto.sum = to_split_int(distribution_update.sum)
+  if hasattr(distribution_update, 'buckets') and \
```

Comment:
Please avoid line continuation (wrap everything in `(...)`).

Reply:
will fix it in next revision.